tests/functional/lib/helpers.js

```diff
@@ -1019,6 +1026,62 @@ define([
   }
 
   /**
+   * Check to ensure an element is displayed.
+   *
+   * @param {string} selector
+   * @returns {promise} rejects if test fails
+   */
+  function testElementDisplayed(selector) {
+    return function () {
+      return this.parent
+        .then(visibleByQSA(selector))
+        .findByCssSelector(selector)
+          .isDisplayed()
+          .then(function (isDisplayed) {
+            assert.isTrue(isDisplayed);
+          })
+        .end();
+    };
+  }
+
+  /**
+   * Ensure the element is not displayed
+   *
+   * @param {string} selector
+   * @returns {promise} rejects if element is displayed
+   */
+  function noDisplayedElement(selector) {
```

Comment:
nit: I think having the `test` prefix here, to match formatting of `testElementDisplayed` might be useful.